### DISCUSSION TITLE
TermsOfService buttons are not visible

### DISCUSSION THREAD

**Discussion by SharpLu** [danny-avila/LibreChat#4365] [Q&A]
Hi Team,

Hope you are doing well!
I just configured the librechat  termsOfService for version LibreChat v0.7.5-rc2

But the pop up window shows something like this.

![image (2)](https://github.com/user-attachments/assets/2a6969a4-6d28-4afe-b787-d81807c0365d)

When i move mouse pointer to the windows it shows the accept or not accept
![Screenshot 2024-10-09 at 13 58 22](https://github.com/user-attachments/assets/ed6299dd-2ae6-48f0-a7db-c29eed223899)
![Screenshot 2024-10-09 at 13 58 31](https://github.com/user-attachments/assets/e5ba4f1e-270d-4ac4-8806-cd173baaca08)

I have attached my config below, any suggestions?

# # For adding a custom config yaml-file you can set the contents in this var
  configYamlContent: |
    includedTools: []
    
    # Enable privacy and terms
    interface:
      # Privacy policy settings
      privacyPolicy:
        externalUrl: 'https://xxx.com/privacy-policy'
        openNewTab: true

      # Terms of service
      termsOfService:
        externalUrl: 'https://xxx.com/tos'
        openNewTab: true
        modalAcceptance: true
        modalTitle: "Terms of Service for LibreChat"
        modalContent: |
          # Terms and Conditions for LibreChat
        endpointsMenu: true
        modelSelect: true
        parameters: true
        sidePanel: true
        presets: true
        prompts: true
        bookmarks: true
        multiConvo: true

**Comment by danny-avila** [comment]
This is already resolved on the latest dev build

**Comment by SharpLu** [comment]
Thank you ! then i will try the latest dev build.

**Comment by SharpLu** [comment]
Hi @danny-avila  Can i follow up a relevant question? about the **privacyPolicy** and **termsOfService**

I have manually created two html for the privacyPolicy and termsOfService and copied the file to docker container /app/ folder
when trying to access the http://localhost:3080/privacy.html or http://localhost:3080/.html i got the below error.
Is there have more place need to be changed?  tried both http://localhost:3080/privacy.html  and http://localhost:3080/privacy got the same error.

Unexpected Application Error!
404 Not Found



  privacyPolicy:
    externalUrl: 'http://localhost:3080/privacy.html'
    openNewTab: true

  # Terms of service
  termsOfService:
    externalUrl: 'http://localhost:3080/.html'
